build_stage3-bolt.bash

```diff
@@ -44,13 +42,9 @@ echo "Optimizing Clang with the generated profile"
 ${BOLTPATH}/llvm-bolt ${CPATH}/clang-15 \
     -o ${CPATH}/clang-15.bolt \
     --data ${TOPLEV}/clang-15.fdata \
-    -reorder-blocks=cache+ \
-    -reorder-functions=hfsort+ \
-    -split-functions=3 \
-    -split-all-cold \
-    -dyno-stats \
-    -icf=1 \
-    -use-gnu-stack || (echo "Could not optimize binary for clang-15"; exit 1)
+	-relocs -split-functions=3 -split-all-cold -icf=1 -lite=1 \
```

Comment:
please keep each option on a line and have the align consistent with `--data`

Reply:
Alright, will be fixed